common/sessionhandler.py

```diff
@@ -55,3 +63,4 @@ def __del__(self) -> None:
         log.debug("Killing session:")
         kill = self.session.get(url=self.del_url)
         log.debug(str(kill) + "\n")
+        del self
```

Comment:
I don't think you need this as it no longer has a `__del__` and it will be GC'd after exiting anyway.